src/app/components/RecentTrades/RecentTradeRow.tsx

```diff
@@ -31,30 +30,26 @@ export const RecentTradeRow: React.FC<RecentTradeRowProps> = ({
     [isOddRow],
   );
 
-  const isLong = useMemo(() => row.loanToken === quoteTokenAddress, [
-    row.loanToken,
-    quoteTokenAddress,
-  ]);
+  const isLong = useMemo(
+    () => row.loanToken.id.toLowerCase() === quoteTokenAddress.toLowerCase(),
+    [row.loanToken, quoteTokenAddress],
+  );
 
   const entryPrice = useMemo(() => {
     return isLong
-      ? bignumber(1)
-          .div(row.entryPrice)
-          .mul(10 ** 36)
+      ? bignumber(1).div(row.entryPrice).toString()
```

Comment:
if for some reason ``row.entryPrice`` is zero then ``entryPrice`` would be infinity

Reply:
Hm, I dont think it's possible, but any suggestions on handling infinities? @DistributedCollective/circle-of-light